_pages/resources.md

```diff
@@ -1,23 +1,47 @@
 ---
 layout: doc
-title: Getting Help
+title: Resources
 ---
 
-## Peer Tutoring
+# Resources
 
-We hold peer tutoring sessions in the basement of Siebel in the computer labs. You can join by adding yourself onto the [Queue](https://edu.cs.illinois.edu/queue/) (the schedule can be found below).
+**If you added late, check [this page]({% link _pages/late_add.md %}) to get caught up.**
 
-Please note that this is not CS 225, so [we]({% link _pages/staff_hall_of_fame.html %}) will not do your assignments. This is one of the last classes you will take before being thrown into 400-level courses, where the assignments are much harder, help is much more limited, and hand holding is negative.
+An introduction to system programming is Angrave's [CS 241 Crowd-Sourced Wikibook](https://github.com/angrave/SystemProgramming/wiki).
```

Comment:
can we rearrange the info here to better indicate that the coursebook is the official supported "book" for the class?

Reply:
Sure. Technically though, the previous wikibook was enough for the course for many semesters, so we should provide a way to let students know that there are options, even if one of them is no longer supported.